src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -102,12 +103,12 @@ function ProductGallery({ title, searchTerm }: Props) {
             <div data-fs-product-listing-pagination="top">
               <GatsbySeo defer linkTags={[{ rel: 'prev', href: prev.link }]} />
               <ButtonLink
-                onClick={(e) => {
+                onClick={(e: MouseEvent<HTMLElement>) => {
```

Comment:
should we add this to the Next.js project as well for the sake of keeping the pattern?